src/Login.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import "./App.css";
+import CreateLobbyLink from "./CreateLobbyLink";
+
+interface ILogin {}
+
+export const checkLogin = async () => {
+  console.log("logging in");
+  const response = await fetch("/loggedIn");
+  const json = await response.json();
+  return json;
+};
+
+const Login = (props: ILogin) => {
+  const [email, setEmail] = React.useState<string>("");
+  const [password, setPassword] = React.useState<string>("");
```

Comment:
Just because having `password` and `state` on the same line sends my blood pressure through the roof, `passwordInput` and `setPasswordInput` makes the intent a bit clearer (we're managing the input values in state, not saved and sensitive user data)